internal/core/custom_resources/resources/layer_attachment.go

```diff
@@ -0,0 +1,103 @@
+package resources
+
+/**
+ * Panther is a Cloud-Native SIEM for the Modern Security Team.
+ * Copyright (C) 2020 Panther Labs Inc
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/aws/aws-lambda-go/cfn"
+	"github.com/aws/aws-sdk-go/service/lambda"
+	"go.uber.org/zap"
+)
+
+const latest = "LATEST"
+
+type LayerAttachmentProperties struct {
+	FunctionArn *string
+	LayerArns   []*string
+}
+
+func customLayerAttachment(_ context.Context, event cfn.Event) (string, map[string]interface{}, error) {
+	switch event.RequestType {
+	case cfn.RequestCreate, cfn.RequestUpdate:
+		return handleCreateUpdateRequests(event)
+	case cfn.RequestDelete:
+		return event.PhysicalResourceID, nil, nil
+	default:
+		return "", nil, fmt.Errorf("unknown request type %s", event.RequestType)
```

Comment:
Since the error is essentially impossible, you can just do

```suggestion
	default:
		// ignore deletes
		return event.PhysicalResourceID, nil, nil
```

Either way is fine, I don't think I've always been consistent here